tools/clang/unittests/HLSL/Objects.cpp

```diff
@@ -700,7 +642,7 @@ TEST_F(ObjectTest, DeclareLocalObject) {
   for (const auto &sod : ShaderObjectData) {
     // When shader models are validated, run through all of them.
     // for (const auto &st : sod.ValidShaderTypes)
-    const auto st = std::first(sod.ValidShaderTypes);
+    // const auto st = std::first(sod.ValidShaderTypes);
```

Comment:
commented out instead of removed since they seemed somewhat related to, though in conflict with the for loop just above which is also commented out.